cvetools/image.go

```diff
@@ -112,7 +114,7 @@ func (s *ScanTools) LoadLocalImage(ctx context.Context, repository, tag, imgPath
 	histories, err := rt.GetImageHistory(imageName)
 	if err != nil {
 		log.WithFields(log.Fields{"error": err}).Error("Failed to get local image history")
-		if err == dockerclient.ErrImageNotFound {
+		if err == ErrImageNotFound {
```

Comment:
Please refer below line to apply the error check.

https://github.com/moby/moby/blob/ca0910c1b903c8dd437dd24fd9a68d92dee17a2c/client/image_inspect_test.go#L36C18-L36C55

Reply:
Thanks for reminding.